ivy/func_wrapper.py

```diff
@@ -1000,18 +1000,23 @@ def _wrap_function(
             if (hasattr(original, attr) and not hasattr(to_wrap, attr)) or (
                 mixed_fn
                 and (
-                    attr == "inputs_to_native_arrays" or attr == "outputs_to_ivy_arrays"
+                    attr == "inputs_to_native_arrays"
```

Comment:
Can we make this function a bit cleaner? Seems like we're making a lot of checks and there's scope to improve readability. Thanks!

Reply:
Updated it. Please let me know if there's anything else that should be changed :+1: :)